tests/fail/tree_borrows/parent_write_invalidates_read.rs

```diff
@@ -0,0 +1,14 @@
+//@compile-flags: -Zmiri-tree-borrows
+
+// Read to local variable kills reborrows *and* raw pointers derived from them.
```

Comment:
"read *from* local variable"

Reply:
The code does a write though, not a read...

Reply:
Oh no, copy-pasted too eagerly